tensorflow_hub/module_v2.py

```diff
@@ -19,6 +19,7 @@
 from tensorflow_hub import native_module
 from tensorflow_hub import registry
 
+DISABLE_CERT_VALIDATION = False
```

Comment:
Let's avoid using this global variable that is also exposed as a public symbol by the library and use an environment variable instead. We don't expect this functionality to be very frequently used. The env variable also allows one to set this behavior more easily from training environments such as TFX.